### PR TITLE
Add Freeciv icon

### PR DESCRIPTION
**Pull request by dash102**

### CODE REVIEW

File: games-strategy/freeciv/freeciv-2.5.5.recipe [hunk 1]
```diff
@@ -99,6 +101,18 @@ INSTALL()
 
 	mkdir -p $appsDir/Freeciv
 	mv $binDir/freeciv-sdl $appsDir/Freeciv/Freeciv
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local INTERNAL="`echo "$portVersion" | cut -d. -f4`"
```
Comment (fbrosson) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: You don't want
```
	local INTERNAL="`echo "$portVersion" | cut -d. -f4`"
```
but
```
	local INTERNAL="0"
```
or
```
	local INTERNAL=0
```
(both work, it's just a question of style.)

Comment (fbrosson) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: Maybe we could hard-code the 0 for internal.
(Most pacakges only have x.y.z versions.)

File: games-strategy/freeciv/freeciv-2.5.5.recipe [hunk 2]
```diff
@@ -99,6 +101,18 @@ INSTALL()
...
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@INTERNAL@|$INTERNAL|" \
+		$portDir/additional-files/freeciv.rdef.in > freeciv.rdef
+
```
Comment (dash102) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: Whoops

File: games-strategy/freeciv/freeciv-2.5.5.recipe (hunk 1 above)
Comment (dash102) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: Do I still keep the -e "s|@INTERNAL@|$INTERNAL|"\ ?

**Comment by dash102** [comment]
Any opinions on the shadow of the icon?

File: games-strategy/freeciv/freeciv-2.5.5.recipe [hunk 3]
```diff
@@ -12,9 +12,11 @@ you do not need to own Civilization to play Freeciv."
 HOMEPAGE="http://www.freeciv.org"
 COPYRIGHT="2002-2015 The Freeciv Team"
 LICENSE="GNU GPL v2"
+REVISION="1"
```
Comment (Begasus) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: This could be set to REVISION="2" as it's an updated recipe

Comment (fbrosson) [inline games-strategy/freeciv/freeciv-2.5.5.recipe]: Good catch!

**Comment by dash102** [comment]
Made a new icon that is closer to the original logo.

File: games-strategy/freeciv/additional-files/freeciv.rdef.in [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	$"3A0A04244C24513651364C0A044F3D4F434852484C0A062451244C322E562E56"
+	$"3449510A043C3D3F3D35253225080F3429382A3E294028422745274829442A42"
+	$"2B412C3F2F3E303C323A33383406138AAA8A8E0030283B413E39343C333F3041"
+	$"2E422C442B452A47482947284427C1E627C0EB27413B29383634080549575455"
+	$"5E405B3D543E0D0A0B0107000A080103301F2401178400040A0301002026180A"
+	$"0201002031270A0101002038180A000100201F270A040101201F240A05010120"
+	$"31240A060102201F240A0701022026150A0801061001178300040A090104201F"
+	$"240A0A010500"
+};
+
```
Comment (Begasus) [inline games-strategy/freeciv/additional-files/freeciv.rdef.in]: rdef file isn't updated, it still shows the first icon you created Stephanie, the iom file is correct

**Pull request merged**